app/views/categories/index.html.erb

```diff
@@ -0,0 +1,39 @@
+<nav>
+  <%= link_to "home", user_categories_path(@user) %>
+  <h3>categories</h3>
+  <% if user_signed_in? %>
+    <%= link_to "Logout", sign_out_user_path, method: :get %>
+  <% end %>
+</nav>
+<main>
+
+  <% if @categories.blank? %>
+    <h3 class="empty">Your categories list is currently empty!</h3>
+    <%= link_to "Add new category", new_user_category_path(@user), class: "empty_link" %>
+  <% else %>
+    <ul id="list" class="list">
+      <% @categories.each do |c| %>
+        <li class="list_item">
+          <%= link_to user_category_activities_path(@user, c) do %>
+            <div class="item-wrapper">
+              <div class="photo">
+                <% if c.icon.attached? %>
+                  <%= image_tag url_for(c.icon), class: "profile_img" %>
+                <% end %>
+
+              </div>
+              <div class="user_name">
+                <p class="muted"><%= c.name %></p>
+                <p class="muted date"><%= c.created_at %></p>
+              </div>
+            </div>
+            <p class="amount">$<%= c.activities.sum { |a| a.amount } %></p>
+          <% end %>
+        </li>
+      <% end %>
+    </ul>
```

Comment:
- [ ] Kindly make sure that the background color for the categories and transactions pages is not white (the color bordering the cards should cover the full page) and it should cover the two pages respectively.

![color](https://github.com/Kaghenimbale/budget-app/assets/53124548/f8d5c693-cf68-4616-afaa-d26a24c8df6e)

![background](https://github.com/Kaghenimbale/budget-app/assets/53124548/8ab1b7b7-883a-481d-94e7-276c296b0ac6)

Reply:
Fixed.